openapi/api.yaml

```diff
@@ -189,6 +237,24 @@ components:
       in: header
       name: X-Xsrf-Token
   schemas:
+    GetIntentResponse:
+      properties:
```

Comment:
Could you take a look at each of the properties in these new schemas and check if any of them are required / will always be provided?

If so, you can mark them as required. This will help with the TypeScript.

Example

![image](https://github.com/user-attachments/assets/0f0ee437-3fdb-4cfb-9a43-57213fcabba6)

Reply:
Great idea! I marked fields "required" as needed. 🙂